lib/include/Tensile/GranularitySelection.hpp

```diff
@@ -0,0 +1,191 @@
+/**
```

Comment:
This set of table / entry / library classes is kind of extraneous since we need to make use of the problem and solution classes directly.  It also prevents sending in the Hardware object in order to get the number of CUs.

Reply:
I will have to rethink this a little bit

Reply:
The last commit addresses this.